core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/AntennaImpl.java

```diff
@@ -22,11 +27,12 @@
 
     private File propertiesFilePath;
 
-    public AntennaImpl(String pluginDescendantArtifactIdName, Path pomFilePath) {
-        this(pluginDescendantArtifactIdName, pomFilePath, null);
+    public AntennaImpl(String pluginDescendantArtifactIdName, Path pomFilePath, Project gradleProject) {
+        this(pluginDescendantArtifactIdName, pomFilePath, gradleProject,null);
     }
 
-    public AntennaImpl(String pluginDescendantArtifactIdName, Path pomFilePath, Path propertiesFilePath) {
+    public AntennaImpl(String pluginDescendantArtifactIdName, Path pomFilePath, Project gradleProject, Path propertiesFilePath) {
+        WrappedGradleProject wrappedGradleProject;
```

Comment:
maybe create function `private WrappedGradleProject getWrappedGradleProject(Path pomFilePath, Project gradleProject, Path propertiesFilePath);` to reduce the size of this function and to get rid of this mutable variable.